src/commands/set.js

```diff
@@ -20,16 +20,21 @@ import { writeJSONSync } from '../utils/fs';
 import { createCommand } from '../utils/helpers';
 import liskAPIInstance from '../utils/api';
 
-const description = `Sets configuration <variable> to <value>. Variables available: json, name, testnet. Configuration is persisted in \`${configFilePath}\`.
+const availableVariables = CONFIG_VARIABLES.join(', ');
+const description = `Sets configuration <variable> to <value>. Variables available: ${availableVariables}. Configuration is persisted in \`${configFilePath}\`.
 
 	Examples:
 	- set json true
 	- set name my_custom_lisky
+	- set api.testnet true
 `;
 
 const WRITE_FAIL_WARNING =
 	'Config file could not be written: your changes will not be persisted.';
 
+const writePropertyWarning = property =>
+	`Config file could not be written: property ${property} does not exist.`;
```

Comment:
We should probably inform the user explicitly that the file is messed up. Something like
```
Config file could not be written: property ${property} was not found. It looks like your configuration file is corrupted. Please check the file or run `lisky clean` to remove it (a fresh default configuration file will be created when you run Lisky again.
```